nin/backend/watch.js

```diff
@@ -29,12 +29,21 @@ function watch(projectPath, cb) {
    * trigger intial callbacks for all files */
   watcher.on('add', function(){ });
 
+  var graphAlreadyLoaded = false;
+
   watcher.on('all', function (event, path) {
     if (event === 'unlink') event = 'delete';
     if (event == 'addDir') {
       return;
     }
 
+    if(path == 'res/graph.json') {
```

Comment:
I'm not sure if an issue is the right tool, but we can add a prominent warning the README, perhaps.